src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.functionality.TaskList;
+import duke.task.Task;
+
+/**
+ * Represents the commands inputted by a user. The Command class cannot be instantiated as it is an abstract class.
+ */
+public abstract class Command {
```

Comment:
Good use of Abstract Class, was wondering why not use an interface instead? It would save you having to add nulls to the attributes in all your commands.